### PR TITLE
Add dummy process into frontend

### PR DESCRIPTION
**Pull request by Felixxxxxxxxxxx**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,7 +1,92 @@
+import './style.css'
+
 function App() {
   return (
     <div className="App">
-      npro
+        <div>npro</div>
```
Comment (jamarob) [inline frontend/src/App.js:6]: `<h1>`?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,7 +1,92 @@
...
+        <div className="parent">
```
Comment (jamarob) [inline frontend/src/App.js:7]: use `<section>` for semantic grouping

File: frontend/src/style.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    width: 100px;
+    height: 50px;
+}
+
+.parent {
+    display: grid;
+    grid-template-columns: repeat(11, 1fr);
+    grid-template-rows: repeat(6, 1fr);
+    grid-column-gap: 0px;
+    grid-row-gap: 0px;
```
Comment (jamarob) [inline frontend/src/style.css:14]: not necessary, 0 is the default

**Review by jamarob — COMMENTED** [review]
lgtm

File: frontend/src/style.css (hunk 3 above)
Comment (Felixxxxxxxxxxx) [reply frontend/src/style.css:14]: removed

File: frontend/src/App.js (hunk 1 above)
Comment (Felixxxxxxxxxxx) [reply frontend/src/App.js:6]: added <h1>

File: frontend/src/App.js (hunk 2 above)
Comment (Felixxxxxxxxxxx) [reply frontend/src/App.js:7]: changed <div> to <section>

**Review by Felixxxxxxxxxxx — COMMENTED** [review]
comments will be adapted with next PR

**Pull request merged**